src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -234,6 +232,8 @@ case class CsvRelation protected[spark] (
           escape = escapeVal,
           commentMarker = commentChar).parseLine(firstLine)
       } else {
+        val escapeVal = if (escape == null) '\\' else escape.charValue()
```

Comment:
Ooops sorry, this might have gotten out of my attention. Removed in next commit.